api/v1alpha1/performanceprofile_conversion.go

```diff
@@ -0,0 +1,120 @@
+package v1alpha1
+
+import (
+	"github.com/openshift-kni/performance-addon-operators/api/v1"
+	"sigs.k8s.io/controller-runtime/pkg/conversion"
+)
+
+// ConvertTo converts this PerformanceProfile to the Hub version (v1).
+func (curr *PerformanceProfile) ConvertTo(dstRaw conversion.Hub) error {
```

Comment:
this has to be manually maintained? needs to be tested (e2e), we need to prevent obscure bugs about fields lost or mangled in conversion

Reply:
yes, we need to manually convert between APIs field by field. It needs both a unit-test and an e2e test. I'll add the unti-test as part of this PR, and add an2e2 test as part of v1->v2 conversion PR in which we have both the regular fields and a "special" case.
Or maybe I'll add an e2e here anyway and just improve it for v2.

Reply:
Added an e2e test to this PR